xilem/examples/calc.rs

```diff
@@ -192,6 +192,7 @@ fn app_logic(data: &mut Calculator) -> impl WidgetView<Calculator> {
             nums.map(|num| digit_button(num).flex(1.)),
             operator_button(operator).flex(1.),
         ))
+        .flex(1.0)
```

Comment:
I would add this to the individual items, rather than inside the creation function.
That matches the existing items; the vertical `flex` is a property of the container the row is in, not inherent to the item itself.

Reply:
You're probably right...